### ISSUE TITLE
Conflicts with Evernote Web Clipper extension

### ISSUE THREAD

**Issue by Altonymous** [rfletcher/safari-json-formatter#21]

**Comment by rfletcher** [comment]
I've been using it with Safari 6 betas for months, and now with 6.0 in Mountain Lion, and I haven't noticed any issues.

**Comment by Altonymous** [comment]
Here's showing it installed...
- http://d.pr/i/3wYB

Here's the test page you setup
- http://d.pr/i/vf32

I have restarted Safari.  Is there anything else I need to do?

**Comment by rfletcher** [comment]
Maybe it's conflicting with another extension you've installed.  Could you try disabling other extensions and seeing if that fixes it?  (And if so, turning them back on one by one to see which one is the culprit.)

**Comment by Altonymous** [comment]
Good call.  It's Evernote Web Clipper.  Try installing that and see if you have the same issue.

**Comment by rfletcher** [comment]
Thanks.  I'll take a look.
